pkg/utils/constant.go

```diff
@@ -0,0 +1,8 @@
+package utils
+
+const (
+	PortNo    = ":8080"
+	HMDB      = "HMDB"
+	HMDB_TEST = "HMDB_TEST"
+	Doctors   = "doctorsCollection"
```

Comment:
Please understand that `Doctors` is a generic word, variable name should DoctorCollection and not its value. Value should be doctors.